Makefile

```diff
@@ -59,6 +58,10 @@ nuke: ## Stop all containers, delete all containers, volumes, and configuration
 	@make down
 	@./tools/scripts/nuke.sh
 
+switch: ## Switch to the specified oc project (n=environment)
```

Comment:
nice way to not have to remember the cryptic namespace names.  `make switch n=dev`